go/libraries/doltcore/sqle/integration_test/database_revision_test.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2022 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration_test
```

Comment:
These are fine but they're probably more easily expressed as script tests to be run by the engine test framework

Reply:
It might be hard to access the commit history inside of the engine test framework, the commit hashes are not static